homeassistant/util/yaml.py

```diff
@@ -119,10 +120,15 @@ def _env_var_yaml(loader, node):
         raise HomeAssistantError(node.value)
 
 
+def _secret_yaml(loader, node):
+    """Load secrets and embed it into the configuration YAML."""
+    return secrets.get_secret(node.value) or ''
```

Comment:
Another added benefit of this is that we will not keep one dictionary with all the user secrets. That's something that will one day backfire on us.